dev/com.ibm.ws.com.unboundid/src/com/ibm/ws/com/unboundid/InMemoryLDAPServer.java

```diff
@@ -47,37 +57,66 @@
     private InMemoryDirectoryServerConfig config = null;
     private InMemoryDirectoryServer ds = null;
 
+    private static final String keystorePassword = "LDAPpassword";
+    private String keystore;
+
     /**
      * Creates a new instance of the in memory LDAP server. It initializes the directory
```

Comment:
This javadoc is wrong, even though you marked the conversation as resolved. Move this Javadoc to the other constructor.